commons/src/main/com/mbrlabs/mundus/commons/Scene.java

```diff
@@ -158,7 +163,7 @@ public void render(float delta) {
         modelCacheManager.update(delta);
 
         if (sceneGraph.isContainsWater()) {
-            captureDepth(delta);
+            if (Gdx.gl30 == null) captureDepth(delta);
```

Comment:
Minor thing: You can use Gdx.graphics.isGL30Available(). Maybe this is a bit more elegant then a null check.

Reply:
Yeah, it does the exact same thing (checks if gl30 is null) but its alot more verbose. I guess the readability is better though

Reply:
Ok, made the change. I also changed it so that Scene uses Nestable FBOs as well. When I originally added these in, I only  needed them on the EditorScene due to editor using FBOs. But, for those wanting to do post processing at runtime, we should use NestableFrameBuffer on Scene as well and now don't have to update FBO code in two places.